gatsby/sourceNodes.js

```diff
@@ -0,0 +1,46 @@
+const fetch = require('node-fetch')
+
+module.exports = exports.sourceNodes = async ({ actions, createContentDigest, createNodeId }) => {
+    const { createNode } = actions
+
+    const integrations = await fetch(
+        'https://raw.githubusercontent.com/PostHog/integrations-repository/main/integrations.json'
+    ).then((res) => res.json())
+    integrations.forEach((integration) => {
+        const { name, url, ...other } = integration
+        const node = {
+            id: createNodeId(`integration-${name}`),
+            parent: null,
+            children: [],
+            internal: {
+                type: `Integration`,
+                contentDigest: createContentDigest(integration),
+            },
+            url: url.replace('https://posthog.com', ''),
+            name,
+            ...other,
+        }
+        createNode(node)
+    })
+
+    const plugins = await fetch(
+        'https://raw.githubusercontent.com/PostHog/plugin-repository/main/repository.json'
```

Comment:
I've now updated the integrations repo to also contain the updated plugins.json, so we can pull both from the same place for simplicity.

```suggestion
        'https://raw.githubusercontent.com/PostHog/integrations-repository/main/plugins.json'
```